hieradata/dev/wso2/wso2am/1.10.0/mesos/default.yaml

```diff
@@ -34,7 +34,8 @@ wso2::mgt_hostname: marathon-lb.marathon.mesos
 wso2::clustering:
   enabled: true
   membership_scheme: mesos
-#  mesos:
+  mesos:
+    marathon_applications: wso2am-gateway-manager,wso2am-gateway-worker
```

Comment:
Yes, but if we move the clustering section to the default.yaml file, the same configuration would apply to default profile. Which would not be correct. Let me explain how this works.

These marathon application names are used by the membership scheme for initializing the hazelcast cluster. In worker/manager separated deployment pattern, we need to let all members in worker cluster and manager cluster to join the same hazelcast cluster. Therefore we specify both worker and manager application names in this property. In the default profile we do not want that to happen since there would be only one cluster (default). Therefore if we move this configuration section to the default.yaml, clustering in default profile would not work.